app/js/utils/globalProperty.js

```diff
@@ -21,12 +22,12 @@ export const APP_GLOBAL_PROPERTIES = {
   labResultsTestLocationQuestion: "labworkflowowa.locationOfLaboratory",
   labResultsEstimatedCollectionDateQuestion: "labworkflowowa.estimatedCollectionDateQuestion",
   labResultsEstimatedCollectionDateAnswer: "labworkflowowa.estimatedCollectionDateAnswer",
-  labResultsToDisplayConceptSet: "labworkflowowa.labResultsToDisplayConceptSet",
   dateAndTimeFormat: "labworkflowowa.dateAndTimeFormat",
   enableLabelPrinting: "labworkflowowa.enableLabelPrinting",
   labelPrintingEndpoint: "labworkflowowa.labelPrintingEndpoint",
   ordersBatchSize: "labworkflowowa.ordersBatchSize",
   orderLabTestLink: "labworkflowowa.orderLabTestLink",
+  labCategoriesConceptSet: "labworkflowowa.labCategoriesConceptSet",
```

Comment:
The new labCategoriesConceptSet GP you added points to the same UUID as the labOrderablesConceptSet GP below. I think that you could safely remove the labOrderablesConceptSet GP and use the new one you just introduced.

Reply:
Yeah, where is the labOrderablesConceptSet used in the labworkflow owa @mogoodrich .  Do we still need this?

Reply:
@mseaton , a few places in code have to be updated if we get rid of labOrderablesConceptSet. Here is the beginning of that refactoring:
https://github.com/openmrs/openmrs-owa-labworkflow/blob/master/app/js/actions/labOrderablesAction.js#L34

Reply:
Yep, good point guys... originally I had replaced the lab orderables GP with the new GP, but then released the orderables was being used elsewhere, namely when generating the test types categories on the list of lab orders page, which I was thinking maybe *should* be based on the orderables?

![2022-08-11_14-17](https://user-images.githubusercontent.com/1163015/184211008-7c61ad76-adbf-46a3-a586-fbd00ed9708e.png)

Anyway, my plan was to ticket that in a separate ticket, and/or make sure we handle it when/if we change the way lab orderables work?